app/gui/src/main/scala/org/bitcoins/gui/WalletGUIModel.scala

```diff
@@ -33,6 +35,7 @@ class WalletGUIModel(dlcModel: DLCPaneModel)(implicit system: ActorSystem) {
     override def run(): Unit = {
       Platform.runLater {
         updateBalance()
+        updateWalletAccounting()
         updateWalletInfo()
         dlcModel.updateDLCs()
```

Comment:
Might be worth filing an issue to batch some of these things into a single command. `updateWalletAccounting` and 
`updateDLCs`are both calling `listDLCs`, it would be nice if we could put these together

Reply:
#3234